fury/actor.py

```diff
@@ -1759,6 +1759,83 @@ def cone(centers, directions, colors, heights=1., resolution=10,
     return actor
 
 
+def octagonalprism(centers, directions=(1, 0, 0), colors=(1, 0, 0), scale=(1, 1, 1)):
+    """Visualize one or many octagonal prisms with different features.
+
+    Parameters
+    ----------
+    centers : ndarray, shape (N, 3)
+        Octagonal prism positions
+    directions : ndarray, shape (N, 3)
+        The orientation vector of the octagonal prism.
+    colors : ndarray (N,3) or (N, 4) or tuple (3,) or tuple (4,)
+        RGB or RGBA (for opacity) R, G, B and A should be at the range [0, 1]
+    scale : int or ndarray (N,3) or tuple (3,), optional
+        Octagonal prism size on each direction (x, y), default(1)
+
+    Returns
+    -------
+    vtkActor
+
+    Examples
+    --------
+    >>> from fury import window, actor
+    >>> scene = window.Scene()
+    >>> centers = np.random.rand(5, 3)
+    >>> dirs = np.random.rand(5, 3)
+    >>> heights = np.random.rand(5)
+    >>> actor = actor.octagonalprism(centers, dirs, (1, 1, 1), heights=heights)
+    >>> scene.add(actor)
```

Comment:
The doc example has `heights` instead of `scale` thus its giving this error: `TypeError: octagonalprism() got an unexpected keyword argument 'heights'`

Reply:
I think it's better to rename it as `heights` as it is consistent with the other actors.